Assets/Scripts/LevelCompletionZone.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using Events;
+using UnityEngine;
+
+[RequireComponent(typeof(BoxCollider))]
+public class LevelCompletionZone : MonoBehaviour
```

Comment:
Similar comment to the DeathZone, should try and give it a namespace and put it in a sub-folder for scripts.

Reply:
ai